http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1237,6 +1237,45 @@ paths:
             - openid
             - email
             - profile
+  "/api/google/v1/apps/{googleProject}/batchNodepoolCreate":
```

Comment:
I like the term "batch" create as it conveys the full meaning without reference to any other information. I know we've been referring to it as "pre"create for historical reasons but "pre" is a relative term and I wonder if it's better to not have any such relativity as this endpoint is not concerned with how those batch created nodepools are utilized. If agreed, it'd be nice to use that term consistently across the implementation, too.

Reply:
what's the relevant between `pre` and `batch` here?

Reply:
Sorry, I should've made the comment stand-alone instead of inline. I was referring to the seemingly interchangeable usage of the terms _pre_ and _batch_ throughout the code, not specifically to the line above.